taskcoordinator/app/src/main/resources/application.properties

```diff
@@ -6,10 +6,13 @@ spring.datasource.password=123
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.hibernate.ddl-auto=validate
 spring.datasource.initialization-mode=always
+
 spring.jpa.properties.hibernate.default_schema=${database.schema.name}
 spring.flyway.schemas=${database.schema.name}
+spring.flyway.placeholders.schema=${database.schema.name}
 
 # kafka properties
+kafka.activateConsumers=true
```

Comment:
Please add really useful and comprehensive comments for each peoperty. I want the meaning if each property to be absolutely clear for everyone